src/clj/game/cards/programs.clj

```diff
@@ -696,6 +696,37 @@
                             :effect (effect (draw :runner 1))
                             :msg "draw 1 card"}}}
 
+   "RNG Key"
+   {:events {:pre-access-card {:req (req (get-in card [:special :rng-guess]))
+                               :delayed-completion true
+                               :msg (msg "to reveal " (:title target))
+                               :effect (req (if-let [guess (get-in card [:special :rng-guess])]
+                                              (if (or (= guess (:cost target))
+                                                        (= guess (:advancementcost target)))
+                                                (continue-ability state side
+                                                                  {:prompt "Choose RNG Key award"
+                                                                   :delayed-completion true
```

Comment:
Because neither path of this ability's effect causes a delayable action, the ability doesn't need delayed completion.